WaysofWork.md

```diff
@@ -224,49 +225,51 @@ To be developed
 
-## Meetings & Co-working
+## Ceremonies: Meetings & Co-working
 
-Synchronous meeting has been minimised according to team's preference and due to the already meeting intensive schedule,
-therefore the focus is in maintaining up to date repositories and communicating in advance via written reports.
+Synchronous meeting has been minimised according to the team's preference and due to the already meeting-intensive schedule.
+The focus is on maintaining up-to-date repositories and communicating in advance via written reports.
 
 Meeting time should be focus on collaboration and group decisions.
 
-### Weekly meeting
+### Weekly team meeting
 Key facts
+
 - Mondays 10.00 to 10.30
 - Attended by Project working team
-- Purpose:
-  - communicate work done and plans (Async)
-  - address blockers
-  - plan the week and co-working time
-  - (If needed) reasign work
-  - (If needed) Contribute discussion points for Monthly meeting, or elevate decisions
-
-These meetings are the main way for the team to coordinate,
+- Agenda:
+  - Asynchronous items
+    - What we've done this week
+    - Things that are blocking me
+    - Coworking session suggestions
+  - Synchronous items
+    - Identify blockers and arrange to solve. Do not spend time fixing problems in this meeting
+    - Write plans for the week. Meeting organisers leads a group discussion on task prioritisation.
+
+These meetings are the main way for the team to coordinate and structured around active stories,
 they are attended by all team members accountable for carrying out the work to deliver the project.
 
-It's asynchronous elements are as important as the actual meeting time and require everyone to update their issues and stories.
+Its asynchronous elements are as important as the actual meeting time and require everyone to update their issues and stories.
+It is by filling the weekly meeting that team members report about their work, for this reason it is key to provide comprenhensive updates that the RPM can take to stories for reporting.
+This way each team member needs to provide updates in a single place.
 
-The weekly meeting is the time to discuss and agree any new work or reprioritise existing work.
-This is not expected to happen each week,
-yet it is likely that after monthly meetings or main milestones (quarters, end of funding) this becomes the focus of the meeting.
-It is during this meeting that any inclussion of issues in the backlog is discussed (and elevated to the Monthly when necessary), 
-and when issues are taken from the backlog into planned work.
 
 ### Weekly coworking
 Key facts
+
+
 - Wednesdays 09.30 to 10.30
 - Attended by Project working team
-- Purpose: work together in issues identified during weekly
+- Purpose: work together in issues identified during weekly or which arise during the week
 
-Co-working time is an unstructured hour for synchronous work
+Co-working time is an unstructured hour for synchronous work. It has proved 
```

Comment:
It has proved what?

Reply:
```suggestion
Co-working time is an unstructured hour for synchronous work.
```

Reply:
I made this point somewhere else and forgot to take out my half done sentence